incubating/pipeline-trigger-merge/step.yaml

```diff
@@ -0,0 +1,65 @@
+kind: step-type
+version: '1.0'
+metadata:
+  name: laurent-cf/pipeline_trigger_merge
+  version: 1.0.0
+  isPublic: false
+  description: >-
+    Merge a pipeline spec and a list of triggers (files or directories) to
+    create a codefresh full pipeline.
+    Directories are processed recursively. Unknown files are simply ignored.
+  sources:
+    - >-
+      https://github.com/codefresh-io/steps/tree/master/incubating/pipeline-trigger-merge
+  stage: incubating
+  maintainers:
+    - name: Laurent Rochette
+    - email: laurent.rochette@codefresh.io
+  categories:
+    - utilities
+  official: false
+  tags: []
+  examples:
+    - description: pipeline-trigger-merge
+      workflow:
+        MergeTriggersIntoPipeline:
+          title: "Merge the triggers into the pipeline spec"
+          type: laurent-cf/pipeline_trigger_merge
+          arguments:
+            SPEC: spec.yml
+            TRIGGERS: trig1.yml TRIGS trig2.yml
+spec:
+  arguments: |-
+    {
+      "definitions": {},
+      "$schema": "http://json-schema.org/draft-07/schema#",
+      "type": "object",
+      "additionalProperties": false,
+      "patterns": [],
+      "required": [
+        "SPEC"
+      ],
+      "properties": {
+        "SPEC": {
+          "type": "string",
+          "description": "The name of the file containing the pipeline spec."
+        },
+        "TRIGGERS": {
+          "type": "string",
+          "description": "A space separated list of trigger files and/or folders."
+        }
+      }
+    }
+  stepsTemplate: |-
+    pipeline-trigger-merge:
+      name: pipeline-trigger-merge
+      image: lrochette/ptm:1.0
```

Comment:
Also missing revision from semver of line 56

Reply:
what do you mean by semver? I changed it to 1.0.0 to match the metadata.version
I think it would be better to use a variable as we have to set it up on line 56 and 5 if possible